Helpers.ps1

```diff
@@ -384,7 +383,7 @@ function Write-Log
                 }
                 else
                 {
-                    $logFileMessage | Out-File -FilePath $Path -Append
+                    $logFileMessage | Out-File -FilePath $Path -Append -WhatIf:$false -Confirm:$false
```

Comment:
Why do we need `$Confirm:$false`?

```suggestion
                    $logFileMessage | Out-File -FilePath $Path -Append -WhatIf:$false
```

Reply:
Because otherwise if `Confirm` has been specified in a calling function, `Out-File` will re-prompt. I tested this to be 100% sure.